partner_identification/views/res_partner_view.xml

```diff
@@ -13,8 +13,41 @@
                         colspan="4"
                         nolabel="1"
                         widget="one2many_list"
+                        mode="tree,form"
                         context="{'default_partner_id': id}"
-                    />
+                        >
+                    <form>
```

Comment:
Doesn't the standard behavior load the base form?

https://github.com/OCA/partner-contact/blob/227da0a6e88f16a7b325b3e662ebfdf6a7155a5c/partner_identification/views/res_partner_id_number_view.xml#L7

Reply:
@vehi-invitu maybe it's only the missing display-block in the chatter that does not show the chatter in the popup
BTW I like your view reorganisation, can you apply it to the base form view ?